Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1/types.go

```diff
@@ -2597,7 +2597,19 @@ type SecurityContextConstraints struct {
 
 	// AllowPrivilegedContainer determines if a container can request to be run as privileged.
 	AllowPrivilegedContainer bool `json:"allowPrivilegedContainer" description:"allow containers to run as privileged"`
+	// Initial capabilities are defaulted into containers.
+	// Initial capabilities will be enforced with the following rules
+	// 1.  InitialCapabilities.Drop and AllowedCapabilities cannot conflict (you may not require
+	//		a cap to be dropped but also let it be requested)
+	// 2.  Capabilities in InitialCapabilities.Add will be added to the container unless the container specifically
+	// 		requests it be dropped.
+	// 3.  Capabilities in InitialCapabilities.Drop will be dropped from the container
+	// 4.  Containers that request to add a capability that is required to be dropped will fail
+	//		validation.
+	InitialCapabilities Capabilities `json:"initialCapabilities" description:"capabilities that will be added or dropped"`
```

Comment:
This name makes it seem like capabilities can change over time. "initial" needs a time context to be understood, and it's a little odd because the sequence is: 1) pod spec submitted to the API may specify capabilities, 2) admission drops some capabilities, adds some capabilities unless dropped by the pod, 3) object is created and capabilities can never be changed.

What about "assignedCapabilities"? They are what admission will assign to the pod at creation time, and the pod author can explicitly opt out of some of the added capabilities by dropping them in their submitted pod spec.